exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -37,10 +39,23 @@
 public class FragmentWrapper {
   private final MajorFragmentProfile major;
   private final long start;
+  private static Integer runningProfileProgressThreshold = new Integer(0);
```

Comment:
Please do not use static, just make variable final and assign it in the constructor.

Reply:
Done.